python-package/lightgbm/sklearn.py

```diff
@@ -597,7 +597,9 @@ def _process_params(self, stage: str) -> Dict[str, Any]:
         if callable(self._objective):
             if stage == "fit":
                 self._fobj = _ObjectiveFunctionWrapper(self._objective)
```

Comment:
We don't need `self._fobj` anymore.
```suggestion
```

P.S. GitHub doesn't allow to make suggestion for the piece of code where lines were both removed and added, so my suggestion is in a form of three consecutive comments.

Reply:
copy that